frontends/web/src/routes/account/send/components/inputs/scan-qr.module.css

```
@@ -0,0 +1,243 @@
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+  background: var(--scan-qr-control-background);
+}
+.controls img,
+.controls svg {
+  position: relative;
+  width: 18px;
+  height: 18px;
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)